### PR TITLE
Add a section in installation docs about running NEST with EBRAINS

### PR DESCRIPTION
**Pull request by jessica-mitchell**
This PR, as discussed in docathon with @clinssen @steffengraber @heplesser @ddahmen , adds the try it on ebrains button for an installation free option in the install section.

Note that the links directly takes the person to a one neuron example (assuming they are logged and have server running). The reason: all the examples will be loaded, and the kernel that contains NEST should be set. So less changes for user. 
We would need to add additional instructions if we want to open a plain notebook on EBRAINS, which is another possibility.

Missing from the instructions so far (wip in another branch, but can be added here) is this issues that you have to basically click twice to load an example if you dont have a server running yet.  

#3066 #3067 should be merged first!

See [output for Read the Docs here](https://nest-simulator--3121.org.readthedocs.build/en/3121/installation/index.html)

### CODE REVIEW

File: doc/htmldoc/index.rst [hunk 1]
```diff
@@ -1,16 +1,162 @@
...
+   synapse_models[<img src='_static/img/synapse.svg' /> \n Synapse Models]
+   stimulating_devices[<img src='_static/img/stimulatelight.svg' /> Stimulating Devices]
+   recording_devices[<img src='_static/img/recordinglight.svg' /> Recording Devices]
+   network[<img src='_static/img/networkbrainlight.svg' /> \n Network Models]
+   click network href "./networks/spatially_structured_networks.html"
+   click neuron_models href "./models/index_neuron.html"
+   click synapse_models href "./models/index_synapse.html"
+   click stimulating_devices href "./models/index_generator.html"
+   click recording_devices href "./models/index_device.html"
+   click simulate href "./nest_behavior/running_simulations.html"
```
Comment (heplesser) [inline doc/htmldoc/index.rst:120]: This part should go for the time being, until we have figured out the figure, shouldn't it?
```suggestion
```

File: doc/htmldoc/installation/cmake_options.rst [hunk 2]
```diff
@@ -104,7 +104,7 @@ Build documentation
 If either documentation build is toggled to `ON`, you can then run ``make docs`` if you only want to
 build the docs.
 
-See also the :ref:`documentation workflow <doc_workflow>` for user and developer docs.
+See also the :ref:`documentation workflow <doc_workflow>` for user facing and technical docs.
```
Comment (heplesser) [inline doc/htmldoc/installation/cmake_options.rst:107]:
```suggestion
See also the :ref:`documentation workflow <doc_workflow>` for user-facing and technical docs.
```

File: doc/htmldoc/installation/cmake_options.rst [hunk 3]
```diff
@@ -180,7 +180,7 @@ Generic build configuration
 | ``-Dwith-intel-compiler-flags=[OFF|<list;of;flags>]``| User defined flags for the Intel compiler                        |
 |                                                      | [default='-fp-model strict']. Separate multiple flags by ';'.    |
 +------------------------------------------------------+------------------------------------------------------------------+
-| ``-Dwith-cpp-std=[<C++ standard>]``                  | C++ standard to use for compilation [default='c++17'].           |
+| ``-Dwith-cpp-std=[<C++ standard>]``                  | C++ standard to use for compilation [default='c++11'].           |
```
Comment (heplesser) [inline doc/htmldoc/installation/cmake_options.rst:183]: Surprised to see c++11 here, we use c++17 now.

```suggestion
| ``-Dwith-cpp-std=[<C++ standard>]``                  | C++ standard to use for compilation [default='c++17'].           |
```

File: doc/htmldoc/installation/index.rst [hunk 4]
```diff
@@ -3,40 +3,69 @@
...
-    .. grid-item-card:: |dev| Install NEST from source
-       :class-title: sd-d-flex-row sd-align-minor-center
+      .. rst-class:: imgbutton
 
-       I'm a developer who wants to :ref:`do development in NEST <dev_install>`
+         .. image:: https://nest-simulator.org/TryItOnEBRAINS.png
+            :target: https://lab.ebrains.eu/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2Fnest%2Fnest-simulator-examples&urlpath=lab%2Ftree%2Fnest-simulator-examples%2Fnotebooks%2Fnotebooks%2Fone_neuron.ipynb&branch=main
 
-.. grid:: 1 1 2 2
+      For more info see :ref:`our guide to running notebooks  <run_jupyter>`
```
Comment (heplesser) [inline doc/htmldoc/installation/index.rst:42]:
```suggestion
      For more info see :ref:`our guide to running Jupyter notebooks on EBRAINS  <run_jupyter>`
```

**Review by heplesser — CHANGES_REQUESTED** [review]
@jessica-mitchell Looks good, just a few small suggestions.

File: doc/htmldoc/installation/hpc_install.rst [hunk 5]
```diff
@@ -3,10 +3,24 @@
 High Performance Computer Systems Installation
```
Comment (steffengraber) [inline doc/htmldoc/installation/hpc_install.rst:3]: Is this file linked somewhere?

File: doc/htmldoc/installation/index.rst [hunk 6]
```diff
@@ -3,40 +3,69 @@
...
+      :class-title: sd-d-flex-row sd-align-minor-center
+      :columns: 4
 
-       I'm an admin or user who wants to :ref:`run NEST on HPC <admin_install>`
+      With the JupyterHub service from EBRAINS you can
+      try out NEST (and many other tools) directly in your browser.
 
-    .. grid-item-card:: |dev| Install NEST from source
-       :class-title: sd-d-flex-row sd-align-minor-center
+      .. rst-class:: imgbutton
```
Comment (steffengraber) [inline doc/htmldoc/installation/index.rst:38]: Aren't the public NEST desktop installations also options for using NEST installation-free?
Hifis: https://nest-desktop.helmholtz-cloud.desy.de
Ebrains: https://nest-desktop.apps.hbp.eu

File: doc/htmldoc/installation/user.rst [hunk 7]
```diff
@@ -32,6 +32,8 @@ We have live media (.ova) if you want to run NEST in a virtual machine.
 Linux |linux|
```
Comment (steffengraber) [inline doc/htmldoc/installation/user.rst:32]: Is this file still linked?

**Review by steffengraber — COMMENTED** [review]
Overall a good step forward for the documentation.
Only have a few comments.

File: doc/htmldoc/installation/index.rst (hunk 6 above)
Comment (jessica-mitchell) [reply doc/htmldoc/installation/index.rst:38]: do we want to directly link to NEST desktop applications? I was thinking of directing users to the NEST desktop documentation, and let users discover the available ways of using NEST desktop through their site

File: doc/htmldoc/installation/docker.rst [hunk 8]
```diff
@@ -37,20 +38,24 @@ like this:
...
 
 .. code-block:: bash
 
     wget https://raw.githubusercontent.com/nest/nest-docker/master/docker-compose.yml
 
 You can then run ``docker-compose up`` with one of the following options:
 
-- NEST server
+NEST server
+~~~~~~~~~~~
```
[the remark this answers is not in the document]
Comment (jessica-mitchell) [reply doc/htmldoc/installation/docker.rst:58]: the way I normally do it is in this order  `===` `---` `~~~` `^^^^`. I htink I was trying to be more consistent and insure the headlines were more visible

File: doc/htmldoc/installation/hpc_install.rst (hunk 5 above)
Comment (jessica-mitchell) [reply doc/htmldoc/installation/hpc_install.rst:3]: It's removed in #3067

File: doc/htmldoc/installation/user.rst (hunk 7 above)
Comment (jessica-mitchell) [reply doc/htmldoc/installation/user.rst:32]: Yes, just not the top reference, subsections are referenced

File: doc/htmldoc/installation/index.rst (hunk 6 above)
Comment (steffengraber) [reply doc/htmldoc/installation/index.rst:38]: It's not necessary to link directly, but these things can be mentioned. But that is not a stop for the PR.

**Review by steffengraber — APPROVED** [review]
+1

**Review by heplesser — APPROVED** [review]

**Pull request merged**